src/rpc/api/mock/validator.ts

```diff
@@ -0,0 +1,66 @@
+import {Attestation, AttestationData, BeaconBlock, bytes32, Deposit, Shard, Slot, Eth1Data} from "../../../types/index";
+
+import {getEmptyBlock} from "../../../chain/genesis";
+
+import {IValidatorApi} from "../interfaces/index";
+import {bytes, bytes48, Fork, number64, SyncingStatus, uint64, ValidatorDuty} from "../../../types";
+
+export interface MockAPIOpts {
+  head?: BeaconBlock;
+  pendingAttestations?: Attestation[];
+  getPendingDeposits?: Deposit[];
+  Eth1Data?: Eth1Data;
+  attestationData?: AttestationData;
+}
+
+export class MockAPI implements IValidatorApi {
```

Comment:
Is there a reason to not move Mock api to test directories?

Reply:
Not reason -- was just following previous code structure tbh. Happy to move it